crates/router/src/routes/app.rs

```diff
@@ -1,10 +1,17 @@
 use std::sync::Arc;
 
 use actix_web::{web, Scope};
+#[cfg(all(
+    feature = "olap",
+    any(feature = "hashicorp-vault", feature = "aws_kms")
+))]
+use analytics::AnalyticsConfig;
```

Comment:
Do we need these imports?

Reply:
My bad, I recently update the base branch for merging, and didn't remove these.